src/shared/components/TimeSeries.tsx

```diff
@@ -267,6 +269,7 @@ class TimeSeries extends Component<Props, State> {
 
       let statuses = [] as StatusRow[][]
       if (check) {
+        // DLW FIXME TODO: decide how to handle
         const extern = buildUsedVarsOption(
           queries.map(query => query.text),
```

Comment:
This is multiple queries.
Some of which may define the variable internally. Some of which may not.
So how do we handle when/if to include the windowPeriod variable?